pkg/restapi/vc/operation/operations_test.go

```diff
@@ -195,9 +184,60 @@ const validVC = `{` +
     "type": "CredentialStatusList2017"
   }
 }`
+	// VC without issuer
+	invalidVC = `{` +
+		validContext + `,
+  "id": "http://example.edu/credentials/1872",
+  "type": "VerifiableCredential",
+  "credentialSubject": {
+    "id": "did:example:ebfeb1f712ebc6f1c276e12ec21"
+  },
+  "issuanceDate": "2010-01-01T19:23:24Z"
+}`
 
-//nolint:lll
```

Comment:
I noticed there were some super long lines in some of these consts, so I split them up into multiple lines. This way we don't need that `nolint`